ganalytics.cpp

```diff
@@ -112,11 +113,12 @@ GAnalytics::Private::Private(GAnalytics *parent)
     screenResolution = getScreenResolution();
 #endif // QT_GUI_LIB
     request.setHeader(QNetworkRequest::ContentTypeHeader, "application/x-www-form-urlencoded");
-    appName = QCoreApplication::instance()->applicationName();
-    appVersion = QCoreApplication::instance()->applicationVersion();
+    appName = "";
```

Comment:
revert these removals, it should use default values, and will be overwritten using public method `generateUserAgent()`